doc/manuals/orion-api.md

```diff
@@ -1340,9 +1383,18 @@ The values that `options` parameter can have for this specific request are:
 | `keyValues` | When used, the response payload uses the `keyValues` simplified entity representation. See [Simplified Entity Representation](#simplified-entity-representation) section for details. |
 | `append`    | Force an append operation.                                                                                                                         |
 
-**Request payload**
+_**Request headers**_
 
-Content-Type is `application/json`
+| Header               | Optional | Description                                                                                    | Example            |
+|----------------------|----------|------------------------------------------------------------------------------------------------|--------------------|
+| `Content-Type`       |          | MIME type. Required to be `application/json`.                                                  | `application/json` |
+| `Fiware-Service`     | ✓        | Tenant or service. See subsection [Multitency](#multitenancy) for more information.            | `acme`             |
+| `Fiware-ServicePath` | ✓        | Service path or subservice. See subsection [Service Path](#service-path) for more information. | `project`          |
+
+_**Request payload**_
+
+The request payload is an object with the attribute information following the JSON representation for attributes described in 
+[JSON Attribute Representation](#json-attribute-representation) and [Partial Representations](#partial-representations) section.
```

Comment:
The /attrs operations are not using the attribute payload described "JSON Attribute Representation" but entity representation without `id` and `type`, more in this style:

> The response is an object representing the entity identified by the ID. The object follows
the JSON entity representation format (described in [JSON Entity Representation](#json-entity-representation) section and
side [Simplified Entity Representation](#simplified-entity-representation) and [Partial Representations](#partial-representations) sections),
but omitting `id` and `type` fields.

Reply:
Note that the description specifies that it just contains **attribute information**, since it is an update.

Reply:
Resolved after a Skype call